flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -66,4 +66,9 @@ public OperatorStateBackend createOperatorStateBackend(
 			String operatorIdentifier) throws Exception {
 		return new DefaultOperatorStateBackend(env.getUserClassLoader());
 	}
+
+	@Override
+	public void close() throws Exception {
+
+	}
```

Comment:
Does it make sense to not implement the `close` method here? Then all subclasses would be forced to think about what happens when closing the state backend?